homeassistant/components/vicare/binary_sensor.py

```diff
@@ -232,3 +256,22 @@ def update(self):
             _LOGGER.error("Vicare API rate limit exceeded: %s", limit_exception)
         except PyViCareInvalidDataError as invalid_data_exception:
             _LOGGER.error("Invalid data from Vicare server: %s", invalid_data_exception)
+
+
+class ViCareStatusBinarySensor(ViCareBinarySensor):
```

Comment:
I am guessing you made this a separate class to make sure its always available, but when I look at `ViCareBinarySensor`, available is only False when `self._attr_is_on is not None` so would the trick be to make sure the `value_getter` function never returns `None`?

Reply:
I think `available` should consider the `isOnline` status of the device config in the end.